test/experiments/KiD_driver/run_KiD_simulation.jl

```diff
@@ -104,6 +104,33 @@ function run_KiD_simulation(::Type{FT}, opts) where {FT}
             ),
             coord,
         )
+    elseif precipitation_choice == "PrecipitationP3"
+        # commenting here the configuration which has been used for p3:
+        # config["precipitation_choice"] = "PrecipitationP3"
+        # config["moisture_choice"] = "MoistureP3"
+        # config["n_elem"] = 24
+        # config["z_max"] = 3000
+        # config["t_end"] = 75
+        # config["w1"] = 0
+        # config["rv_0"] = 0
+        # config["rv_1"] = 0
+        # config["rv_2"] = 0
+        # config["dt"] = Float64(0.5)
+        cloudy_params = nothing
+        init = map(
+            coord -> CO.p3_initial_condition(
```

Comment:
why do we need a special case for the P3 scheme? Could it be handled in the same way as other bulk microphysics cases?

Reply:
I think if we want to initialize particles with specific P3 properties, we have to have another initial condition function, and in that initial condition, we specify our own temperature and pressure gradients, and we can add vapor pressure, etc. In the future we could try other initial conditions, but maybe for now this way is easiest for testing if you agree?